.github/scripts/ci-test.sh

```diff
@@ -7,45 +7,45 @@ cd repos/jikesrvm
 # Only run one test
 
 # RBaseBaseNoGC
-python scripts/testMMTk.py -j $JAVA_HOME -g RBaseBaseNoGC -a "Xms1024M Xmx1024M" -- --use-third-party-heap=../../ --use-third-party-build-configs=../../jikesrvm/build/configs --use-external-source=../../jikesrvm/rvm/src
+python scripts/testMMTk.py -j $JAVA_HOME -g RBaseBaseNoGC -a "Xms1024M Xmx1024M" -- --use-third-party-heap=../../ --use-third-party-build-configs=../../jikesrvm/build/configs --use-external-source=../../jikesrvm/rvm/src --m32
```

Comment:
I doubt if we can upstream this 'testMMTk.py' script. Maybe we should not be using it (at some point).